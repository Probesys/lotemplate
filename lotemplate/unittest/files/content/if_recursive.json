{
  "tutu": {
    "type": "text",
    "value": "oui"
  },
  "tata": {
		"type": "text",
		"value": "foo"
  }
}