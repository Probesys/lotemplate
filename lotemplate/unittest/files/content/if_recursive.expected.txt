
Tests of recursive if statements


  
displayed
  
  






This paragraph should be displayed (!= glouglou)

several lines




(Should be displayed without space, 3 ifs)

